src/test/scala/tv/teads/tour/basics/FinalExamSpec.scala

```diff
@@ -0,0 +1,24 @@
+package tv.teads.tour.basics
+
+import org.scalatest.{FlatSpec, Matchers}
+import tv.teads.tour.basics.FinalExam.Bidder
+
+class FinalExamSpec extends FlatSpec with Matchers {
```

Comment:
What about the case of two bidders with highest price ? 😬

Reply:
👍